llpc/translator/lib/SPIRV/libSPIRV/SPIRVType.cpp

```diff
@@ -86,6 +91,32 @@ SPIRVWord SPIRVType::getIntegerBitWidth() const {
   return static_cast<const SPIRVTypeInt *const>(this)->getBitWidth();
 }
 
+unsigned SPIRVType::getSizeInBytes() const {
+  if (isTypeVector())
+    return getVectorComponentType()->getSizeInBytes() * getVectorComponentCount();
+  if (isTypeMatrix())
+    return getMatrixColumnType()->getSizeInBytes() * getMatrixColumnCount();
```

Comment:
it is  better to  add a comment about no alignment is expected.

Reply:
Will add it.